libs/mock-block-dock/src/datastore/use-mock-datastore/non-temporal.ts

```diff
@@ -137,7 +137,7 @@ export const useMockDatastore = (
               JSON.stringify(currentGraph),
             ) as Subgraph;
 
-            addEntitiesToSubgraphByMutation(newSubgraph, [newEntity]);
+            addEntityVerticesToSubgraphByMutation(newSubgraph, [newEntity]);
```

Comment:
I think we need to update the edges here. Calling the new edge methods might still be problematic here depending on if someone called `updateEntity` on a different entity before hand. If you check the logic of `updateEntity` maybe you can see what we do with the old edges, if we delete them or update their intervals or whatever.

If we correctly modify the edges with `updateEntity` then calling the new edge method here is probably safe because the `addOutwardEdge...` methods check for equality before adding edges (they won't add exact duplicates), so it _should_ only add new edges implied by this entity

Reply:
Done in e41c98c99a6b89748dc43a0e729ed97aca402b44